React/NextJs/ProjectStructure.md

```diff
@@ -0,0 +1,212 @@
+# Project file structure
+
+## Organizing components
+(For PR Review):
+I believe this section will be refactored once we get around with all chakra stuff and component organization methodology.
+I kept going with only atomic design approach.
+
+### UI Components
+
+Before implementing [atomic design methodology](https://github.com/danilowoz/react-atomic-design), our common practice was separating components by the scope of the page + core components + shared components like this:
```

Comment:
I'm not sure this has any value in the handbook since it will be obsolete and forgotten. Maybe it would be more useful as a comparison once the team starts using it.